source/state_representation/test/tests/space/cartesian/test_cartesian_state.cpp

```diff
@@ -520,6 +520,93 @@ TEST(CartesianStateTest, Truthiness) {
   EXPECT_TRUE(empty);
 }
 
+TEST(CartesianStateTest, TestMultiplicationOperators) {
+  CartesianState state = CartesianState::Random("test");
```

Comment:
This is not gonna work for the reason that CartesianState(test, world)  is not compatible with CartesianPose(test, world), which is why i had them as CartesianState(world, world) initially

Reply:
Good point, of course it was different for the addition operators. It's difficult to catch that behavior when you already expect the tests to fail ;)

Reply:
That is indeed a good point thanks for checking that. I am doing division operator next and will correct it then.